ci/docker/Dockerfile-benchmark

```diff
@@ -0,0 +1,22 @@
+FROM frolvlad/alpine-glibc
+
+CMD ["./benchmarks" "--help"]
+ENV RUNFILES_DIR=/usr/local/bin/benchmarks/benchmarks.runfiles
+ENV ENVOY_PATH=envoy
+ENV ENVOY_IP_TEST_VERSIONS=v4only
+WORKDIR /usr/local/bin/benchmarks
+
+COPY benchmarks /usr/local/bin/benchmarks/
+
+RUN apk add docker openrc
+RUN rc-update add docker boot
+RUN echo "**** install Python ****" && \
+  apk add --no-cache python3 && \
+  if [ ! -e /usr/bin/python ]; then ln -sf python3 /usr/bin/python; fi && \
+  echo "**** install pip ****" && \
+  python3 -m ensurepip && \
+  rm -r /usr/lib/python*/ensurepip && \
+  pip3 install --no-cache --upgrade pip setuptools wheel && \
```

Comment:
I'm also wondering if we should consider pinning versions here?

Reply:
I have an idea of how we can do this.  It does make sense.  I'll test and update.